src/site/content/en/blog/optimizing-content-efficiency-eliminate-downloads/index.md

```diff
@@ -23,9 +25,38 @@ The fastest and best-optimized resource is a resource not sent. You should elimi
+
+### Largest Contentful Paint (LCP)
+
+[Largest Contentful Paint (LCP)](/lcp/) measures when the largest text node, image, or video with a `poster` image loads. It's considered an important perceptual metric that gives the user the impression that a site is loading quickly.
+
+In general, downloading less resources means that the bandwidth the user does have will be allocated across less resources, and may translate to an improvement in LCP. A classic example is that of lazy loading, where images outside of the viewport during page load will not be downloaded until the browser has determined the user is more likely to see them. If you have a large thumbnail gallery of, say, 50 images, lazy loading all but the top ten of them means that the browser can make more efficient use of the bandwidth available to it, and the remaining images will load more quickly.
+
+However, it's not just about loading less _images_, necessarily. While the browser has an internal prioritization scheme that determines how much bandwidth each resource should receive, _all_ resources—particularly those downloaded at high priority—have the potential to deprive a potential LCP element's dependent resource, especially on slow network connections. That dependent resource may be an image file that represent's the pages LCP element—but it could also very well be a web font resource that the browser needs to render a text node that may be determined as the page's LCP element.
+
+If your website is heavy on text, it may be the case that a page's LCP element is a text node. While there are many [good font optimization and loading strategies](/font-best-practices/), it may be worth considering whether a system font is sufficient for your website's needs, so that LCP elements which are text nodes can load without a dependency on a web font resource and paint almost immediately as the CSS and HTML arrives from the server.
+
+### Cumulative Layout Shift (CLS)
+
+Every resource you load that contributes to a page's [Cumulative Layout Shift (CLS)](/cls/) is a resource that you have to manage. For images, it involves practices such as setting explicit dimensions. For fonts, it involves manage font loading and potentially fallback font matching to minimize shifts during a web font's swap period. For JavaScript, it could be managing how that script manipulates the DOM so that layout shifts are reduced to an acceptable amount. In every case, a resource that contributes to a page's CLS requires some amount of work to ensure page layout is sufficiently stable.
```

Comment:
Not loving that first sentence. Every resource, whether it contributes to CLS or not, is a resource you have to manage IMHO. Plus let's link it more to efficient downloads. Also not convinced you need the last sentence (seems to repeat first sentence). Or maybe it should be moved to the start of the next paragraph?

```suggestion
Every resource you load has the potential to contribute to a page's [Cumulative Layout Shift (CLS)](/cls/), particularly if has not finished downloading by the time of the initial paint. For images, avoid CLS involves practices such as setting explicit dimensions. For fonts, managing font loading and potentially fallback font matching can minimize shifts during a web font's swap period. For JavaScript, it could be managing how that script manipulates the DOM so that layout shifts are reduced to an acceptable amount.
```